compiler/cpp/src/thrift/generate/t_java_generator.cc

```diff
@@ -5265,6 +5274,9 @@ THRIFT_REGISTER_GENERATOR(
     "    android_legacy:  Do not use java.io.IOException(throwable) (available for Android 2.3 and "
     "above).\n"
     "    option_type:     Wrap optional fields in an Option type.\n"
+    "    handle_runtime_exceptions:\n"
+    "                     Generated services will handle RuntimeException as "
```

Comment:
I think it'd be hard to properly explain the TTransportException the client will experience otherwise, in a short usage text.